R/ship_reports.R

```diff
@@ -0,0 +1,98 @@
+#' #' Ship Reports
```

Comment:
this file might use some refactoring into smaller components as well - plan is to test it, see if any major changes need to be made before investing in a refactor